archinstall/locales/base.pot

```diff
@@ -652,7 +652,7 @@ msgid ""
 msgstr ""
 
 msgid ""
-"Choose which servers to install, if none then a minimal installation wil be "
+"Choose which servers to install, if none then a minimal installation will be "
```

Comment:
Can you make this correction in all languages? The translated string will be the same

Reply:
Ok